util/common_task.py

```diff
@@ -70,3 +70,5 @@ def deactivate_services(user):
 def generate_corp_group_name(corpname):
     return 'Corp_' + corpname.replace(' ', '_')
 
+def generate_alliance_group_name(alliancename):
+    reutrn 'Alliance_' + alliancename.replace(' ', '_')
```

Comment:
>>> from celerytask.tasks import run_databaseUpdate
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/home/allianceserver/allianceauth/celerytask/tasks.py", line 19, in <module>
    from util.common_task import deactivate_services
  File "/home/allianceserver/allianceauth/util/common_task.py", line 74
    reutrn 'Alliance_' + alliancename.replace(' ', '_')
                     ^
SyntaxError: invalid syntax